### PR TITLE
v1.1

### PR DESCRIPTION
**Pull request by josephmiclaus**

### CODE REVIEW

**Comment by miclaus** [comment]
@doomsayer2 Please let me know when I can merge.

In the meantime, I'll have a look at the leftovers.

File: public/assets/js/constants.js [hunk 1]
```diff
@@ -1,9 +1,6 @@
-/// wait for debug
-// alert('start safari debug');
-
 /// constants
 
-var _live = false;
+var _live = true;
```
Comment (josephmiclaus) [inline public/assets/js/constants.js:3]: **Note** that I've set this version to **LIVE**

**Comment by josephmiclaus** [comment]
@doomsayer2 Could you please test if we are good to go ? :)

**Comment by doomsayer2** [comment]
Doing now as i create the mobidul and all around to test it. Exceptionally great work on the last commits. Great work totally. Impressive.

**Comment by doomsayer2** [comment]
Really great work and nasty bug but awesome done.

File: app/controllers/UserController.php [hunk 2]
```diff
@@ -166,7 +166,8 @@ public function sendFeedback () {
       $feedbackData = [
         'user' => $params->user,
         'code' => $params->code,
-        'feedback' => $params->feedback
+        'feedback' => $params->feedback,
+        'created_at' => date('Y-m-d H:i:s', time())
```
Comment (josephmiclaus) [inline app/controllers/UserController.php:170]: @doomsayer2 Good input ! I'll have a look.

**Review by josephmiclaus — COMMENTED** [review]
Should be ready for merge.

File: app/controllers/UserController.php (hunk 2 above)
Comment (josephmiclaus) [reply app/controllers/UserController.php:170]: @doomsayer2 Works for me ( see below ). 

Remember to run `php artisan migrate`. :) 

screen shot 2016-12-16 at 11 22 41

**Pull request merged**